edge-agent/docker/linux/arm32v7/base/Dockerfile

```diff
@@ -1,18 +1,4 @@
 ARG base_tag=2.1.6-runtime-bionic-arm32v7
-ARG num_procs=4
```

Comment:
remove this since it is built by the pipeline, right?

Reply:
Yep. This was added here to improve build of Rocks DB on container. Since we are doing it out of band (documented on one note), I'm removing this parameter.